finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -49,21 +50,30 @@ public void add(String hostname, Properties systemProps) {
         props.setProperty("user.name", systemProps.getProperty("user.name"));
 
         SystemData system = new SystemData(hostname, props);
+        // tag::Add[]
         if (!systems.contains(system)) {
             // tag::custom-tracer[]
+            // tag::Try[]
+            // tag::addSpan[]
             try (Scope childScope = tracer.buildSpan("add() Span")
+            // end::addSpan[]
                                               .startActive(true)) {
                 // tag::addToInvList[]
                 systems.add(system);
                 // end::addToInvList[]
             }
+            // end::Try[]
             // end::custom-tracer[]
         }
+        // end::Add[]
     }
-
+    // tag::Traced[]
```

Comment:
- add blank line above this line so methods are spaced out properly

Reply:
Addressed.